crates/host/src/wasmbus/mod.rs

```diff
@@ -877,7 +707,7 @@ impl Messaging for Handler {
         }
     }
 
-    #[instrument]
+    #[instrument(skip(self, __arg1))]
```

Comment:
What is this trying to achieve? Maybe just `skip_all`?

Reply:
This skips the `body` field on `BrokerMessage`. Since we're destructuring that arg, I can't do `skip(self, body)`, but `__arg1` allows me to skip `BrokerMessage`. The alternative was not destructuring. Would you prefer that?